src/components/JsPsychExperiment.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Experiment } from "jspsych-react";
```

Comment:
file extension on this and login should be `.jsx` (is that what we did on the tasks?)